### DISCUSSION TITLE
May I ask how to use it?

### DISCUSSION THREAD

**Discussion by zoushucai** [crazy-max/docker-flarum#88] [Q&A]
I looked at the document and felt that it wasn't very detailed, especially for me, a network novice. The first problem I encountered was that I couldn't log in, as shown below:

```
Oops! Something went wrong during a cross-origin request. Please reload the page and try again.

```

Can you help me solve it?

**Comment by qwani** [comment]
same here  im using shared hosting and the admin reluctantly installed docker  
the coomands are not clear for me too
